src/index.ts

```diff
@@ -132,6 +132,7 @@ export {default as RadioGroup, RadioGroupProps} from './components/radioGroup';
 export type {RecorderProps} from './typings/recorderTypes';
 export type {ComponentStatics} from './typings/common';
 export {default as ScrollBar, ScrollBarProps} from './components/scrollBar';
+export {default as SearchInput, SearchInputProps, SearchInputRef, SearchInputPresets} from './components/searchInput';
```

Comment:
SearchInputPresets should not be exported, you access preset via static member 'presets'

Reply:
Let's sync about this, I don't think we should add it to the component statics anymore

Reply:
Ok let's talk about it...

Reply:
removed